UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentAPI.cs

```diff
@@ -358,6 +361,11 @@ static void ProcessAgentResponse(HeartbeatResponse heartBeat)
                 }
             }
 
+            if (OnMaintenanceV2Callback != null && heartBeat.MaintenanceSchedule != null)
+            {
+                OnMaintenanceV2Callback.Invoke(heartBeat.MaintenanceSchedule);
+            }
```

Comment:
Is there a reason null propagation for event invocation is never used in this project? Reduces nesting (in some places more than others), but I noticed it does not really get used. Ex, [here](https://github.com/PlayFab/gsdk/blob/0bf86011f252a0a427e57aaa7f3c8b843a4d49b8/UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentAPI.cs#L357-L360) and [here](https://github.com/PlayFab/gsdk/blob/0bf86011f252a0a427e57aaa7f3c8b843a4d49b8/UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentAPI.cs#L395-L398)

Reply:
This is almost certainly because most of this was probably written before null propagation was available or had to support older Unity versions that did not support that version of C#. And this was only written this way because that was the established pattern in this package. Though in the future, we can certainly make a pass to update old patterns like this.